development.postgresql.yml

```diff
@@ -0,0 +1,72 @@
+version: "2"
+services:
+  server:
+    build:
+      context: .
+      dockerfile: server.Dockerfile
+    networks:
+      serverdb:
+      signer:
+      server:
+        aliases:
+          - notary-server
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-server -config=fixtures/server-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/server/postgresql
+      DB_URL: postgres://server@serverdb:5432/notaryserver?sslmode=disable
+    depends_on:
+      - serverdb
+      - signer
+  signer:
+    build:
+      context: .
+      dockerfile: signer.Dockerfile
+    networks:
+      signerdb:
+      signer:
+        aliases:
+          - notarysigner
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-signer -config=fixtures/signer-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/signer/postgresql
+      DB_URL: postgres://signer@signerdb:5432/notarysigner?sslmode=disable
+    depends_on:
+      - signerdb
+  serverdb:
```

Comment:
If we can agree these compose files are for dev and testing, which they are, then having a single database server instance makes it much more convenient for us to inspect state and even `docker commit` a single running database container to preserve specific states we may want to share for debugging/demonstration purposes.